<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="4">
     <widget class="QListView" name="lvChildGeneration">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="lIterationAmount">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pbCreateGeneration">
      <property name="text">
       <string>Create Generation</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QCustomPlot" name="cp5BestInLastGeneration" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pbShowGraph">
      <property name="text">
       <string>Show Graph</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pbProcess">
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QCustomPlot" name="cp5BestInFirstGeneration" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="4" rowspan="2">
     <widget class="QListView" name="lvNextGeneration">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QCustomPlot" name="cp5BestInMiddleGeneration" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="3">
     <widget class="QListView" name="lvFirstGeneration">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="4" colspan="4">
     <widget class="QCustomPlot" name="cpGraph" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QSlider" name="hsIterationAmount">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pb5BestGraphShow">
      <property name="text">
       <string>5 Best Graphs Show</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
